src/com/owncloud/android/db/UploadDbObject.java

```diff
@@ -0,0 +1,321 @@
+package com.owncloud.android.db;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+import android.accounts.Account;
+import android.content.Context;
+import android.util.Base64;
+
+import com.owncloud.android.authentication.AccountUtils;
+import com.owncloud.android.datamodel.OCFile;
+import com.owncloud.android.db.UploadDbHandler.UploadStatus;
+import com.owncloud.android.files.services.FileUploadService.LocalBehaviour;
+import com.owncloud.android.lib.common.operations.RemoteOperationResult;
+import com.owncloud.android.lib.common.utils.Log_OC;
+
+/**
+ * Stores all information in order to start upload operations. PersistentUploadObject can
+ * be stored persistently by {@link UploadDbHandler}.
+ * 
+ * @author LukeOwncloud
+ * 
+ */
+public class UploadDbObject implements Serializable {
+
+    /** Generated - should be refreshed every time the class changes!! */
+    ;
+    private static final long serialVersionUID = -2306246191385279928L;
+
+    private static final String TAG = "UploadDbObject";
+    
+    public UploadDbObject(OCFile ocFile) {
+        this.ocFile = ocFile;
+    }
+//    /**
+//     * Local path to file which is to be uploaded.
+//     */
+//    String localPath;
+//    /**
+//     * Remote path where file is to be uploaded to.
+//     */
+//    String remotePath;
+//
+//    /**
+//     * Mime type of upload file.
+//     */
+//    String mimeType;
+    OCFile ocFile;
+    
+    public OCFile getOCFile() {
+        return ocFile;
+    }
+    
+    
+    /**
+     * Local action for upload.
+     */
+    LocalBehaviour localAction;
+
+    /**
+     * Date and time when this upload was first requested.
+     */
+    Calendar uploadTime = new GregorianCalendar();
+
+    public Calendar getUploadTime() {
+        return uploadTime;
+    }
+
+    /**
+     * @return the uploadStatus
+     */
+    public UploadStatus getUploadStatus() {
+        return uploadStatus;
+    }
+
+    /**
+     * Sets uploadStatus AND SETS lastResult = null;
+     * @param uploadStatus the uploadStatus to set
+     */
+    public void setUploadStatus(UploadStatus uploadStatus) {
+        this.uploadStatus = uploadStatus;
+        setLastResult(null);
+    }
+
+    /**
+     * @return the lastResult
+     */
+    public RemoteOperationResult getLastResult() {
+        return lastResult;
+    }
+
+    /**
+     * @param lastResult the lastResult to set
+     */
+    public void setLastResult(RemoteOperationResult lastResult) {
+        this.lastResult = lastResult;
+    }
+
+    /**
+     * Overwrite destination file?
+     */
+    boolean forceOverwrite;
```

Comment:
IMHO if you put the properties of a class at the beginning it is easier to read the class. 
We usually use the order:
- Properties
- Constructors
- Getters & Setters

Could you reorder the implementation of this class?